Dockerfile

```diff
@@ -17,25 +20,25 @@ ARG IMAGE_VERSION=""
 
 # Custom build options:
 #  Disable certain image layers using build env variables if desired
-ARG IMAGE_LAYER_SYS_EXT
-ARG IMAGE_LAYER_PERL_EXT
-ARG IMAGE_LAYER_DEV
-ARG IMAGE_LAYER_PERL_CPAN
-ARG IMAGE_LAYER_PERL_CPAN_EXT
-ARG IMAGE_LAYER_PYTHON
-ARG IMAGE_LAYER_PYTHON_EXT
-ARG IMAGE_LAYER_NODEJS
-ARG IMAGE_LAYER_NODEJS_EXT
+ARG IMAGE_LAYER_SYS_EXT="1"
+ARG IMAGE_LAYER_PERL_EXT="1"
+ARG IMAGE_LAYER_DEV="1"
+ARG IMAGE_LAYER_PERL_CPAN="1"
+ARG IMAGE_LAYER_PERL_CPAN_EXT="1"
+ARG IMAGE_LAYER_PYTHON="1"
+ARG IMAGE_LAYER_PYTHON_EXT="1"
+ARG IMAGE_LAYER_NODEJS="1"
+ARG IMAGE_LAYER_NODEJS_EXT="1"
 
 # Custom installation packages
-ARG APT_PKGS
-ARG CPAN_PKGS
-ARG PIP_PKGS
-ARG NPM_PKGS
+ARG APT_PKGS=""
+ARG CPAN_PKGS=""
+ARG PIP_PKGS=""
+ARG NPM_PKGS=""
```

Comment:
Be careful changing these to have a default value as on Travis CI, it wouldn't work the way it is now. May be different on Github Actions, to be verified.

Reply:
Before i assigned the default values, it was not possible to disable CPAN, PIP, NPM and so on, how did this work before?

Reply:
Just to get an idea, how the unset AGRG operated before:

```
buildx call failed with: failed to solve: rpc error: code = Unknown desc = executor failed running [/bin/sh -c if ( [ "${NPM_PKGS}" != "" ] \|\| [ "${IMAGE_LAYER_NODEJS}" != "0" ] \|\| [ "${IMAGE_LAYER_NODEJS_EXT}" != "0" ] ) && ( [ "${ARCH}" == "AMD64" ] \|\| [ "${ARCH}" == "ARMv7" ] \|\| [ "${ARCH}" == "ARM64v8" ] ] \|\| [ ${TARGETPLATFORM} == 'linux/amd64' ] \|\| ${TARGETPLATFORM} == 'linux/arm/v7' ] \|\| ${TARGETPLATFORM} == 'linux/arm64' ] \|\| ); then 
```

My opinion is, that this comes from the Dockerfile itself how it is preprocessed, but may i am wrong here.

Reply:
I set `NPM_PKGS` to `""` becaue it it later tested against `""` and not against `0` as the LAYERS.